src/docs/product/error-monitoring/reprocessing/index.mdx

```diff
@@ -0,0 +1,96 @@
+
+The dialogue behind it asks you two questions:
+
+* _How many events to reprocess_ -- When left empty, all events are reprocessed
+  and potentially move to other (new or existing) issues if the stacktrace
+  changed significantly. When 20 is entered, the 20 most recent events will be
+  reprocessed and the rest of the issue will be left intact.
+
+* _What to do with remaining events_ -- If you choose to reprocess only 20
+  events out of 100, you can choose what to do with the remaining 80 events.
+  The default is to keep them where they are (the issue you're viewing).
+  Alternatively you may choose to delete them.
+
+After hitting _Reprocess Events_, **the issue will be locked and hidden from
+your issue stream while reprocessing is ongoing**. As data is migrated between
+issues we allow no actions such as resolving or assigning the issue to ensure
+data consistency and avoid data loss.
+
+Note that you can still get to the issue manually by searching for
+`is:reprocessing` (removing all other search terms) to get around those
+restrictions if you have to.
+
+## Caveats
+
+The dialogue for starting reprocessing lists a couple of caveats that you
+should really familiarize yourself with and understand before starting.
+
+* **Data glitches.** During reprocessing you may observe temporary data
+  inconsistencies across the entire product. You may observe an event belong to
+  two issues at once, and appear in Discover queries twice.
+
+  Those inconsistencies disappear the second reprocessing is done, however
+  depending on your platform and the amount of events involved this can take a
+  while.
+
+* **Attachment storage needs to be enabled.** If your events come from
+  minidumps or unreal crash reports, you need to make sure to have [attachment
+  storage](/platforms/native/enriching-events/attachments/#crash-reports-and-privacy)
+  enabled. **If the original minidump no longer exists in Sentry, sending an
+  event through reprocessing will cause it to have no stacktrace at all.**
+
+* **Quota applies.** Every event you choose to reprocess will count against
+  your plan's quota a second time. Rate limits and spike protection do not
+  apply.
+
+* **Debug files don't apply immediately.** When uploading debug information
+  files, regardless whether to your own symbol server or to Sentry, they are
+  only applied after one hour after internal caches have expired. Please wait
+  at least 1 hour after uploading DIFs before reprocessing.
+
+* **Alerts do not apply, with caveats.** Issue alerts do not trigger for
+  reprocessed events, and the new events are not subject to data forwarding.
+
+  Metric alerts generally do not trigger either, however are subject to the
+  aforementioned data inconsistencies. If you have created a metric alert with
+  a 24-hour measurement window, and an event you reprocess happened in the last
+  24 hours, it will temporarily be counted twice. This may set off the metric
+  alert.
```

Comment:
```suggestion
  Metric alerts generally do not trigger either; however, they are subject to the
  aforementioned data inconsistencies. If you have created a metric alert with
  a 24-hour measurement window, and an event you reprocess happened in the last
  24 hours, it will temporarily be counted twice. This may trigger a metric
  alert.
```